OpenRA.Game/Scripting/AIScriptContext.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Eluant;
+using OpenRA.Primitives;
+
+namespace OpenRA.Scripting
+{
+	public sealed class AIScriptContext : ScriptContext
+	{
+		public AIScriptContext(World world, IEnumerable<string> scripts) : base(world, null, scripts)
+		{
+		}
+
+		// Called by Activate() in HackyAI.cs
+		public void ActivateAI(string faction_name, string internal_name)
+		{
+			if (FatalErrorOccurred)
+				return;
+
+			using (var activateAI = (LuaFunction)Runtime.Globals["ActivateAI"])
+			{
+				// In the future, we may need to pass more params,
```

Comment:
Style nit: If this is a real todo please use a `// TODO: ` style comment, otherwise i'm not sure this is worth mentioning.